docs/docs/unit-testing.md

```diff
@@ -32,22 +32,23 @@ npm install --save-dev jest babel-jest react-test-renderer identity-obj-proxy 'b
 Because Gatsby handles its own Babel configuration, you will need to manually
 tell Jest to use `babel-jest`. The easiest way to do this is to add a `jest.config.js`. You can set up some useful defaults at the same time:
 
-```json:title=jest.config.js
+```js:title=jest.config.js
```

Comment:
Not sure if you caught this--I'd be shocked if prettier did?--but... you're right!

Reply:
Yeah it was me 😛 Prettier would add parentheses around the block otherwise, eg. 
```(module.exports = {...})```